docker/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM centos:7
+MAINTAINER sqre-admin
+LABEL version="0.2.1" description="LSST DM/SQuaRE GitHub snapshot tools" \
+      name="lsstsqre/github-snapshot"
+
+USER root
+RUN yum update -y && \
+    yum install -y epel-release && yum repolist && \
+    yum install -y git python-pip jq python-devel libffi-devel \
+     openssl-devel gcc && \
+    curl -s \
+    https://packagecloud.io/install/repositories/github/git-lfs/script.rpm.sh  \
+    -o install-git-lfs-repo.sh && \
+    bash ./install-git-lfs-repo.sh && \
+    yum -y install git-lfs && \
+    pip install sqre-github-snapshot && \
```

Comment:
What do you think about installing sqre-github-snapshot from the source of this repo, rather than installing it from `pip`? That way you could make an image and test a container indepdently of releasing to PyPI?